packages/components/addon/components/hds/accordion/item/index.hbs

```diff
@@ -0,0 +1,25 @@
+<Hds::DisclosurePrimitive class={{this.classNames}} @isOpen={{@isOpen}} @toggle={{this.toggle}} ...attributes>
```

Comment:
Would `@onClickToggle=this.onClickToggle` bring it more in line with what we have elsewhere (the disclosure primitive, I think)? The `on` prefix, also speaks about it being an event.

Reply:
Ah, I see we already have an `onClickToggle`, wired with the Button – do we need both? I'm not sure I follow the difference between the two.

Reply:
I used the `@toggle` to change the tracked `@isOpen` state on the item so that I can update the class name when the `onClickToggle` on the button is called.

Is there better naming I could use to make that clearer? Or perhaps some other way to do it that's better?

Reply:
Thanks I updated to remove the related unneeded code.